### PR TITLE
add Homebrew for macOS

### PR DESCRIPTION
**Pull request by retokromer**

### CODE REVIEW

**Comment by retokromer** [comment]
(and align the section 2)

File: README [hunk 1]
```diff
@@ -27,31 +27,39 @@ BPG Image library and utilities
...
+- Only a 64 bit target is supported because x265 needs it for bit
+  depths > 8.
+
+2.3) macOS
+----------
+
+On macOS libpng can be installed by using Homebrew with the following
+command:
+
+  brew install libpng
```
Comment (eek) [inline README]: You mean `libbpg` not `libpng`

**Review by eek — CHANGES_REQUESTED** [review]

**Comment by retokromer** [comment]
Of course, thank you!

File: README (hunk 1 above)
Comment (jhabdas) [reply README]: Yes, I believe they did. `libbpg` is [a valid formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/libbpg.rb) in homebrew core and pulls from the official gzipped tarball. `libpng` is [specified as](https://github.com/Homebrew/homebrew-core/blob/master/Formula/libbpg.rb#L19) a dependency of `libbpg` Rake task.

**Pull request closed without merging**